stixcore/products/level1/quicklook.py

```diff
@@ -1,142 +1,37 @@
 """
 .
 """
-from datetime import datetime, timedelta
 
-import numpy as np
 
-import astropy.units as u
-from astropy.table import unique, vstack
-from astropy.table.table import QTable
 from astropy.time.core import Time
 
-from stixcore.products.common import _get_energies_from_mask
-from stixcore.products.product import BaseProduct
+from stixcore.processing.engineering import raw_to_engineering_product
+from stixcore.products.level0.quicklook import QLProduct as QLProductL0
+from stixcore.tmtc.packets import GenericPacket
 from stixcore.util.logging import get_logger
 
 __all__ = ['QLProduct', 'LightCurve']
 
 logger = get_logger(__name__)
 
 
-class QLProduct(BaseProduct):
-    def __init__(self, service_type, service_subtype, ssid, data, control, **kwargs):
-        """
-        Generic product composed of control and data
-
-        Parameters
-        ----------
-        control : stix_parser.products.quicklook.Control
-            Table containing control information
-        data : stix_parser.products.quicklook.Data
-            Table containing data
-        """
-        self.type = 'ql'
-        self.service_type = service_type
-        self.service_subtype = service_subtype
-        self.ssid = ssid
-        self.control = control
-        self.data = data
-
-        energies = kwargs.get('energies', False)
-        if not energies:
-            energies = self.get_energies()
-        self.energies = energies
+class QLProduct(QLProductL0):
```

Comment:
it's a bit odd to have the baseclass like this maybe we should rename, but not entirely sure why this was done so maybe it's fine?

Reply:
the QLProductL0 is just an import rename. Basically it means that this Level1QLProduct inherits from L0QLProduct